src/core/lib/BasicOrderFulfiller.sol

```diff
@@ -1035,6 +1006,21 @@ contract BasicOrderFulfiller is OrderValidator {
                 OrderFulfilled_consideration_body_offset
             )
 
+            // Set a length of 1 for the offer array.
+            mstore(add(eventDataPtr, 0x80), 1)
+
+            // Write itemType to the SpentItem struct.
+            mstore(add(eventDataPtr, 0xa0), offeredItemType)
+
+            // Copy calldata region with (offerToken, offerIdentifier,
+            // offerAmount) from OrderParameters to (token, identifier,
+            // amount) in SpentItem struct.
+            calldatacopy(
+                add(eventDataPtr, 0xc0),
+                BasicOrder_offerToken_cdPtr,
+                ThreeWords
+            )
```

Comment:
I moved these down here, since it would make it more clear what we are doing in memory and also would require less calculation of the pointer.